www/apps/docs/content/plugins/search/meilisearch.md

```diff
@@ -127,25 +127,32 @@ const plugins = [
   {
     resolve: `medusa-plugin-meilisearch`,
     options: {
-      // other options...
+      config: {
+        host: process.env.MEILISEARCH_HOST,
+        apiKey: process.env.MEILISEARCH_API_KEY,
+      },
       settings: {
         products: {
           indexSettings: {
-            searchableAttributes: [
-              "title", 
+            searchableAttributes: ["title", "description", "variant_sku"],
+            displayedAttributes: [
+              "title",
               "description",
               "variant_sku",
-            ],
-            displayedAttributes: [
-              "id", 
-              "title", 
-              "description", 
-              "variant_sku", 
-              "thumbnail", 
+              "thumbnail",
               "handle",
             ],
           },
           primaryKey: "id",
+          transformer: (product) => ({
```

Comment:
The `transformer` isn't required, did you have to provide it for the plugin to work?

Reply:
Hi @shahednasser yes, I had to provide it, otherwise it wouldn't work.